firmware/application/hw/debounce.cpp

```diff
@@ -25,7 +25,7 @@
 #include "utility.hpp"
 
 uint8_t Debounce::state() {
-    bool v = state_to_report_;
+    uint8_t v = state_to_report_;
```

Comment:
Note to reviewers:  This local variable should have been a uint8_t.  No functional change.